### PR TITLE
Common func to run config tests

### PR DESCRIPTION
**Pull request by PabstMirror**
Adds a simple common function to run tests.
Mostly just config checkers right now (e.g. test_vehicleInventory.sqf for #5168)

- Makes them easier to call.
- Gets them out of `#ifdef DEBUG_MODE_FULL` so they can be used without file patching
- Very basic test pass / fail reporting

Edit: Some actual code changes in this:
- `latitude` loaded from map configs is now reversed (seems to be an arma convention, matches in-game world map display)
This only effects MGRS and advBallistics (ab uses sin/cos, so it will have zero effect)
- Added some map configs (1.72 Malden)
- Removed `Map Grid Warning` warnings (maps configs will probably never be changed)
Warnings now part of the test system

### CODE REVIEW

**Comment by Glowbal** [comment]
I don't like the `dev/` sub directory for tests. I propose to rename it to `test/`.

**Comment by PabstMirror** [comment]
I was thinking it could hold things like debuging functions like 
https://github.com/acemod/ACE3/blob/medical-rewrite/addons/medical/functions/fnc_dev_watchMedicalStats.sqf

but we could keep those in `\functions` and just put test stuff in `\test`

**Comment by PabstMirror** [comment]
I guess these cross the line between debuging and actual unit testing.

I write a lot of debuging and testing code like this, and like to keep it in the repo, but don't want it cluttering up the main code.

So where should these go, and what do we want them named as?

**Comment by Cuel** [comment]
Will tests be included in release pbos?

**Comment by PabstMirror** [comment]
Yes, for example the rearm/reammo tests will be useful for testing compatibility with other mods.

Those tests weren't usable because they are inside of a `#ifdef DEBUG_FULL` block, which isn't accessible on a release build of ace

**Comment by PabstMirror** [comment]
So what do we want to do with these?
- `\dev`
- `\test`
- keep in `\functions`

File: addons/common/functions/fnc_runTests.sqf [hunk 1]
```diff
@@ -0,0 +1,48 @@
+/*
+ * Author: PabstMirror
+ * Run test functions
```
Comment (jonpas) [inline addons/common/functions/fnc_runTests.sqf]: Missing final dot.

File: addons/common/functions/fnc_runTests.sqf [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+ *
+ * Arguments:
+ * 0: Specifc test to run (default is to run all) <STRING><OPTIONAL>
```
Comment (jonpas) [inline addons/common/functions/fnc_runTests.sqf]: `Specific` and `<STRING> (default: #all)`

File: addons/common/functions/fnc_runTests.sqf [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [] call ace_common_fnc_runTests;
```
Comment (jonpas) [inline addons/common/functions/fnc_runTests.sqf]: I think we decided to do those without `;`.

**Pull request merged**